modules/postgres/src/test/scala/doobie/postgres/NotifySuite.scala

```diff
@@ -4,46 +4,50 @@
 
 package doobie.postgres
 
-import cats.effect.{IO, Sync}
+import cats.effect.{IO}
 import cats.syntax.all.*
 import doobie.*
 import doobie.implicits.*
 import org.postgresql.PGNotification
 
-class NotifySuite extends munit.FunSuite {
+class NotifySuite extends munit.CatsEffectSuite {
   import FC.{commit, delay}
-  import cats.effect.unsafe.implicits.global
   import PostgresTestTransactor.xa
 
   // Listen on the given channel, notify on another connection
   def listen[A](channel: String, notify: ConnectionIO[A]): IO[List[PGNotification]] =
-    (PHC.pgListen(channel) *> commit *>
-      delay { Thread.sleep(50) } *>
-      Sync[ConnectionIO].delay(notify.transact(xa).unsafeRunSync()) *>
-      delay { Thread.sleep(50) } *>
-      PHC.pgGetNotifications).transact(xa)
+    WeakAsync.liftIO[ConnectionIO].use { liftIO =>
+      (for {
+        _ <- PHC.pgListen(channel)
+        _ <- commit
+        _ <- delay { Thread.sleep(50) }
```

Comment:
I wonder if we can also rewrite these:
```suggestion
        _ <- liftIO(IO.sleep(50.millis))
```

Reply:
Idk, but when I am rewriting them as specified, it is giving me errors.

Reply:
In Cats Effect 3, LiftIO has been replaced with Sync.
found this on the web

Reply:
It is not possible to do that.

Reply:
> In Cats Effect 3, LiftIO has been replaced with Sync.
> found this on the web

That's not correct ...

Reply:
> What are the errors?

Reply:
> What are the errors?

<img width="1066" alt="Screenshot 2025-03-11 at 4 56 23 AM" src="https://github.com/user-attachments/assets/a0fa718b-e931-4fb0-a78e-8f5eb337edfe" />

Reply:
Sorry, I wrote it too hastily ... it should be `liftIO.liftIO(...)`, like the other ones.

Reply:
"Yes, that's correct! I didn't notice it either."

Reply:
It worked ,
Updating the PR.

Reply:
Got it . I used 
import scala.concurrent.duration._
it should be 
import scala.concurrent.duration.*